src/components/DimensionMenu.js

```diff
@@ -248,7 +247,7 @@ export class DimensionMenu extends Component {
                     {i18n.t(
```

Comment:
this `i18n.t` call should be removed as it does nothing (`getAxisItemLabel` should be used instead, see comment above)

Reply:
Fixed